conversion/conversion.go

```diff
@@ -550,20 +550,41 @@ func WriteSchemaFile(conv *internal.Conv, now time.Time, name string, out *os.Fi
 	// legal Cloud Spanner DDL (Cloud Spanner doesn't currently support comments).
 	// Change 'Comments' to false and 'ProtectIds' to true to write out a
 	// schema file that is legal Cloud Spanner DDL.
```

Comment:
We can maybe merge this comment to down below where we now actually generate a 'legal' Cloud Spanner DDL.